### DISCUSSION TITLE
Kimai1 still supported?

### DISCUSSION THREAD

**Discussion by Sarmingsteiner** [KatjaGlassConsulting/ApprovalBundle#50] [General]
Since we have done a lot of customization in Kimai1, we are hesitating to change to Kimai2.
We are interested in trying the ApprovalBundle plugin, but we should know if there will be any updates for Kimai1.
Recent updates have only been released for Kimai2.

**Comment by kevinpapst** [comment]
> Kiami1, we are hesitating to change to Kimai2.

You are making the pain even worse ...

**Comment by KatjaGlassConsulting** [comment]
Dear @Sarmingsteiner,

yes the version 1 is still supported as it's been used in a company with Kimai1 for now. Nevertheless, the fixes only had been required for 2 and the new functionality was also only requested for version 2, that's why the version 1 had no updates lately.
